src/qt/optionsmodel.cpp

```diff
@@ -90,6 +90,16 @@ void OptionsModel::Init(bool resetSettings)
     // by command-line and show this in the UI.
 
     // Main
+    if (!settings.contains("bPrune"))
+        settings.setValue("bPrune", false);
+    if (!settings.contains("nPruneSize"))
+        settings.setValue("nPruneSize", 2);
```

Comment:
per the other note, if the minimum value `nMinDiskSpace` is moved to `optionsmodel.h`, we can then just use that here too.

Reply:
We can use `3` and the code will set the prune size to `3000` MB.
Line 98 does this logic. (nPruneSizeMB)